src/commands/Minion/craft.ts

```diff
@@ -72,7 +97,13 @@ export default class extends BotCommand {
 		const requiredItems: [string, number][] = Object.entries(Craft.inputItems);
 
 		// Get the base time to craft the item then add on quarter of a second per item to account for banking/etc.
-		const timeToCraftSingleItem = Craft.tickRate * Time.Second * 0.6 + Time.Second / 4;
+
+		let timeToCraftSingleItem = Craft.tickRate * Time.Second * 0.6 + Time.Second / 4;
+
+		const hasDiary = hasFallyHardDiary(msg.author);
+		if ((Craft.bankChest && hasDiary) || msg.author.skillLevel(SkillsEnum.Crafting) >= 99) {
```

Comment:
this will speed everything up if >=99

Reply:
no because only the tannable items have a value for bankChest

Reply:
you wrote `||`

Reply:
I see what happened when I added diary didn't move bracket over